aiohttp/client_reqrep.py

```diff
@@ -572,6 +583,7 @@ def __init__(self, method, url, *,
         self._timer = timer if timer is not None else TimerNoop()
         self._auto_decompress = auto_decompress
         self._cache = {}  # reqired for @reify method decorator
+        self._traces = traces
```

Comment:
Why in ClientRequest traces is public attribute while in ClientResponse it's private?

Reply:
I can answer: `ClientResponse` is a public class visible by user.
The attr should be private.
About request I don't care.

Reply:
Ok, fine for me.

Reply:
Agreed. I switched no to `_traces` in both classes